content/influxdb/v2.0/get-started.md

```diff
@@ -334,6 +334,32 @@ _To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run
        quay.io/influxdb/influxdb:v2.0.4
    ```
 
+### Configure InfluxDB with Docker
+
+To mount an InfluxDB configuration file and use it from within Docker:
+
+1. follow the steps above in ["Persist data outside the InfluxDB container"](#persist-data-outside-the-influxdb-container).
+
+2. Use the command below to generate the default configuration file on the host file system:
+
+    ```console
+    $ docker run \
+        --rm quay.io/influxdb/influxdb:v2.0.4 \
+        influxd print-config > config.yml
+    ```
+
+3. Modify the default configuration, which will now be available under `$PWD`.
+
+4. Then start the InfluxDB container:
+
+   ```console
+   $ docker run -p 8086:8086 \
+         -v $PWD/config.yml:/etc/influxdb2/config.yml:ro \
+         quay.io/influxdb/influxdb:v2.0.4
```

Comment:
The image hosted on `quay.io` doesn't have the nice auto-detection logic for finding config files that have been mounted into `/etc/influxdb2/`. To make it work, this example will either need to:
1. Switch to `influxdb:2.0.4` as the image used
2. Add an env variable to explicitly set the config path:
    ```console
    $ docker run -p 8086:8086 \
           -v $PWD/config.yml:/etc/influxdb2/config.yml:ro \
           -e INFLUXD_CONFIG_PATH=/etc/influxdb2/config.yml \
           quay.io/influxdb/influxdb:v2.0.4
    ```

My vote would be for 1 (and to replace all references to the quay build with references to the DockerHub build)

Reply:
What would be the command to pull from DockerHub as opposed to quay?

Reply:
Ah, is it `docker run --name influxdb -p 8086:8086 influxdb:2.0.4`?

Reply:
Passing `--name` is independent, you just need to swap in `influxdb:2.0.4` as the image reference

Reply:
So it would be:
```console
$ docker run -p 8086:8086 \
        -v $PWD/config.yml:/etc/influxdb2/config.yml:ro \
        influxdb:2.0.4
```